src/components/Header/index.jsx

```diff
@@ -151,7 +151,15 @@ const GordonHeader = ({ onDrawerToggle }) => {
           >
             <MenuIcon className={styles.menu_button_icon} />
           </IconButton>
-
+          &nbsp;&nbsp; &nbsp;&nbsp;
+          <Tab //this is an "invisible" button behind the new logo that links back to the home page
+            className={styles.tab}
+            label=" "
+            component={ForwardNavLink}
+            disableRipple={true}
+            to="/"
+            onChange={(event, value) => setTabIndex(value)}
+          />
```

Comment:
This is not the best way to do this. We should have an actual image tag in the markup that is wrapped in a link, rather than a background image that is covered by an invisible Tab. Also, I suspect this breaks tab highlighting.

Reply:
Got it. I can look into that. I also made it so clicking on the tab resets the tab highlight to Home, which I can keep intact if necessary. I know there is talk of modifying the header tabs, so it may need to be changed